src/components/Tooltip.js

```diff
@@ -0,0 +1,26 @@
+import ReactTooltip from 'react-tooltip';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faQuestionCircle } from '@fortawesome/free-solid-svg-icons';
+import styled from 'styled-components';
+
+const QuestionMark = styled.div`
+  display: inline-block;
+  color: darkgrey;
+  position: absolute;
+  right: -25px;
+  top: 5px;
+`;
+
+const Tooltip = (props) => {
+  const { text } = props;
```

Comment:
I think you could also deconstruct the text directly in the function.

```suggestion
const Tooltip = ({text}) => {
```

Reply:
Ah sorry I interpreted your "looks good to me" as an approval and merged it already 😄

Reply:
Yes no worries, it was just a remark ;)